eng/CredScanSuppression.json

```diff
@@ -7,7 +7,7 @@
                 "%2BLsuqDlN8Us5lp%2FGdyEUMnU1XA4HdXx%2BJUdtkRNr7qI%3D",
                 "fake",
                 "Test12345",
-                "default",
+                "\"default\"",
```

Comment:
Do you know how many places use this placeholder? If there aren't many can we update them to use another placeholder and remove this from the list completely?